src/snyk/common/proxy.ts

```diff
@@ -57,17 +66,24 @@ export function getVsCodeProxy(workspace: IVSCodeWorkspace): string | undefined
   return workspace.getConfiguration<string>('http', 'proxy');
 }
 
-export function getAxiosProxyConfig(workspace: IVSCodeWorkspace): AxiosRequestConfig {
+export function getAxiosConfig(
+  workspace: IVSCodeWorkspace,
+  configuration: IConfiguration,
+  logger: ILog,
+): AxiosRequestConfig {
+  // if proxying, we need to configure getHttpsProxyAgent, else configure getHttpsAgent
+  let agentOptions: HttpsProxyAgent | Agent | undefined = getHttpsProxyAgent(workspace, configuration, logger);
```

Comment:
No necessity for type declaration 👌

Reply:
Unfortunately, it is needed for the assignment in line 76 to work.